src/rest_framework_api_key/permissions.py

```diff
@@ -58,7 +63,18 @@ def has_permission(self, request: HttpRequest, view: typing.Any) -> bool:
     def has_object_permission(
         self, request: HttpRequest, view: typing.Any, obj: AbstractAPIKey
     ) -> bool:
-        return self.has_permission(request, view)
+        if _drf_version < _3_14_0:  # pragma: no cover
+            # Before DRF 3.14.0 (released in Sept 2022), bitwise OR would skip
+            # .has_permision() and only call .has_object_permission(), resulting in
+            # API key permissions not being checked unless we implemented
+            # .has_object_permission().
+            # Since 3.14.0, DRF appropriately checks for both .has_permission() and
+            # .has_object_permission() when checking object permissions of a bitwise OR.
+            # We kept the old redundant behavior to avoid regressions for users who have
+            # not updated their DRF yet.
+            return self.has_permission(request, view)
+
+        return True
```

Comment:
Instead of returning `True`, would it be better to return `super().has_object_permission(request, view, obj)`? I'm pretty sure that the superclass just returns True right now, but in case they ever decide to change or improve that implementation in the future, I would defer to it.

Reply:
Sure, good point. The intent was indeed to return the default behavior of BasePermissionClass, which happens to be `True` now, but could change.

Reply:
I thought of it because I had submitted an alternate patch for the DRF bug that didn't always return `True`. They didn't accept it even though I think it was more correct. Maybe one day someone will change their mind. Probably not, but couldn't hurt to be prepared.